public_html/index.php

```diff
@@ -23,10 +25,12 @@
 }
 // Shuffle and truncate the list
 shuffle($contributors_img_list);
+*/
 
 //Check if there is an event
-$md_github_url = 'https://github.com/sanger-tol/nf-co.re/tree/main/markdown/events';
-$header_btn_url = 'https://nf-co.re/events/rss';
+// these 2 variables not being used any more, but update them to sanger-tol now
+$md_github_url = 'https://github.com/sanger-tol/pipelines-website/tree/main/markdown/events';
+$header_btn_url = 'https://pipelines.tol.sanger.ac.uk/events/rss';
```

Comment:
If they're not used, I'd rather delete them to avoid any confusion later

Reply:
Yes, like others I currently only commented out. We should remove them as well.